server/task_executor_pitr_cutover.go

```diff
@@ -76,6 +80,38 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 		zap.String("original_database", task.Database.Name),
 		zap.String("pitr_database", pitrDatabaseName),
 		zap.String("old_database", pitrOldDatabaseName))
+
+	log.Info("Appending new migration history record...")
```

Comment:
Can we do this instead?

https://github.com/bytebase/bytebase/blob/main/server/task_executor_database_restore.go#L159

Reply:
What I really mean is to use one line instead of 20+ lines (line 101-123) here.

```
targetDriver.ExecuteMigration(ctx, m, "")
```

https://github.com/bytebase/bytebase/blob/main/server/task_executor_database_restore.go#L172

Reply:
Yes I thought about it, and found it not viable.

https://github.com/bytebase/bytebase/blob/2ba7499c2be753c090a8cb32bf66731371e59c0c/plugin/db/util/driverutil.go#L129

The function `ExecuteMigration` relies on the fact that the same database has (maybe) different schema before and after `executor.Execute(ctx, statement)`. But in our case, we are swapping two databases, and prev/updated schema should be that of these two databases respectively. So IMO we must deal with the schema change here.

Reply:
But the logic here is doing exactly what ExecuteMigration does - unless we record the previous schema before we do the PITR. For schema drift, that's what a baseline migration does.

Reply:
Per offline discussion, the prev/updated schema is not so relevant in a disaster recovery scenario.
Changed to `ExecuteMigration`.